.hooks/pre-commit.R

```diff
@@ -39,6 +39,11 @@ for (file in current_files$files) {
 if (error_flag) {
   cat("Warning, aborting commit. Unrecognised data files found, please update .gitignore or datafiles_log.csv.\n")
   quit(save = "no", status = 1, runLast = FALSE)
+} else {
+  tidy_output <- tidy_code()
```

Comment:
I've adjusted this in KS4RRs to be a little more sensible in how it works as discussed:
https://github.com/dfe-analytical-services/KS4-ready-reckoners/commit/54bf8e935b8dd601d7a5ea39ff1c5924b6499101